pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -707,11 +726,136 @@ def phase_str(phi: float) -> str:
             if ax_lims[i][0] < 0:
                 ax.axhline(0, **hline_kwargs)
 
+    return (fig_reg if register else None, fig, ch_axes, data)
+
+
+def draw_samples(
+    sampled_seq: SequenceSamples,
+    register: Optional[BaseRegister] = None,
+    sampling_rate: Optional[float] = None,
+    draw_phase_area: bool = False,
+    draw_phase_shifts: bool = False,
+    draw_input: bool = True,
+    draw_modulation: bool = False,
```

Comment:
I don't think the user can have a control over these: 
`draw_samples` is displaying the sample he is providing and only this sample (if he wants to draw the output he should modulate it by himself), otherwise we will end up with user modulating twice their samples.